core/src/main/java/org/apache/druid/data/input/InputSource.java

```diff
@@ -78,6 +78,6 @@
   InputSourceReader reader(
       InputRowSchema inputRowSchema,
       @Nullable InputFormat inputFormat,
-      File temporaryDirectory
+      @Nullable File temporaryDirectory
```

Comment:
Is this nullable now? I think it shouldn't be nullable if it's null only in unit tests.

Reply:
oops, i don't remember adding this so I suspect I did this on accident, will fix